conditional/__init__.py

```diff
@@ -56,3 +62,7 @@ def default_route():
 def zoo():
     from conditional.models.migrate import free_the_zoo
     free_the_zoo(app.config['ZOO_DATABASE_URI'])
+
+
+app.run(host=app.config['IP'], port=app.config['PORT'])
```

Comment:
Yup, had to move stuff around so that LDAP initialized before the app started and I never moved it back.